src/graphql/main/schema.graphql

```diff
@@ -259,6 +259,17 @@ type Coordinates {
   longitude: Float!
 }
 
+type Country {
+  code: CountryCode!
+  id: ID!
```

Comment:
what is the difference between `code` and `id`?

Reply:
assuming there is no difference we can only keep `Id`, and add a graphql comment explaining this is a countryCode. 

`Id` helps with out-of-the-box-caching so this is why we should favor it.

Reply:
No difference at the moment. Code is the correct semantic name but it can work for an ID as well. I believe we do the same thing with the Currency object.

Reply:
I suppose we can get rid of ID since it is not used.

Reply:
Sorry that comment was not initially showing up for me. I will switch to only use id

Reply:
Would you keep the type as NonNullId or CountryCode for the id field

Reply:
I will stick with CountryCode for right now as it is more descriptive